x/utils/savepoint_test.go

```diff
@@ -86,7 +80,7 @@ func TestSavepoint(t *testing.T) {
 	}
 
 	for i, tc := range cases {
-		t.Run(fmt.Sprintf("case-%d", i), func(t *testing.T) {
+		t.Run(fmt.Sprintf("case-%s", i), func(t *testing.T) {
```

Comment:
Please rename `i` to `testName` and use it as the test name. No need for the `case-` prefix.